tests/common/gcp_api/compute_test.py

```diff
@@ -117,6 +117,17 @@ def test_get_firewall_rules_errors(self, name, response, status,
         with self.assertRaises(expected_exception):
             list(self.gce_api_client.get_firewall_rules(self.project_id))
 
+    def test_get_networks(self):
```

Comment:
Please add a test for the error cases as well, like test_get_firewall_rules_errors above.

Reply:
Will do!